packages/compass/src/app/index.js

```diff
@@ -184,34 +181,6 @@ var Application = View.extend({
       }),
       this.queryByHook('layout-container')
     );
-
-    const handleTour = () => {
-      if (app.preferences.showFeatureTour) {
-        this.showTour(false);
-      } else {
-        this.tourClosed();
-      }
-    };
-
-    handleTour();
-  },
-  showTour: function(force) {
-    const TourView = require('./tour');
-    const tourView = new TourView({ force: force });
-    if (tourView.features.length > 0) {
-      tourView.on('close', this.tourClosed.bind(this));
-      this.renderSubview(tourView, this.queryByHook('tour-container'));
-    } else {
-      this.tourClosed();
-    }
-  },
-  tourClosed: function() {
-    app.preferences.unset('showFeatureTour');
-    app.preferences.save({}, {success: () => {
-      if (!app.preferences.showedNetworkOptIn && process.env.HADRON_ISOLATED !== 'true') {
-        ipc.ipcRenderer.emit('window:show-network-optin');
```

Comment:
I think we need to keep this part. It shows a settings window after closing the tour.

Reply:
Re-added 👍